react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,32 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (args.length > 0) {
+    if (args.length > 1) {
+      return Math.min(...args);
+    }
+    if (isArray(args[0])) {
+      return Math.min(...args[0]);
+    }
+    return Math.min(args[0]);
+  }
+  return undefined;
 }
 
-export function copy() {
+export function copy(param) {
+  return Object.assign(param.constructor(), param);
+}
```

Comment:
You can use an arrow function and use the implicit return
```
export const copy = param => Object.assign(param.constructor(), param);
```

Reply:
You can transform this into an arrow function and  use the if ternary ` a ? b : c`  This executes like `if a b else c` 
`export const copy = param => isArray(param) ? [...param] : { ...param };`